### PR TITLE
Bin2Unicode: Fix stringy output

### PR DESCRIPTION
**Pull request by zachthompson**
###### Description of changes

Precedence issues in the string version of the output.  Fixed for html/lite.

---

Instant Answer Page: https://duck.co/ia/view/bin2unicode
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @zachthompson

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/Bin2Unicode.pm [hunk 1]
```diff
@@ -124,7 +124,7 @@ handle matches => sub {
     return if $str =~ /^[\p{Control} ]+$/;
 
     DONE:
-    return "Binary '$bin_string' converted to " . $want_ascii ? 'ascii' : 'unicode' . " is '$str'",
+    return "Binary '$bin_string' converted to " . ($want_ascii ? 'ascii' : 'unicode') . " is '$str'",
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/Bin2Unicode.pm]: Would you be against using 
```perl
"Binary '$bin_string' converted to @{[$want_ascii ? 'ascii' : 'unicode']} is '$str'"
```

?

Comment (zachthompson) [inline lib/DDG/Goodie/Bin2Unicode.pm]: Yeah, unfortunately that refs/derefs for no reason.  The parens are the standard way to signal precedence.

Comment (mintsoft) [inline lib/DDG/Goodie/Bin2Unicode.pm]: I agree with @zachthompson I much prefer the ternary

Comment (GuiltyDolphin) [inline lib/DDG/Goodie/Bin2Unicode.pm]: @zachthompson @mintsoft Sure - I just saw it in the docs once and I do love tapping that `@` key :wink:

In that case - thanks @zachthompson, LGTM!

screenshot-2016-06-08-23 20 07_658_204

I just realised it describes it as 'chicanery' o-o

Comment (zachthompson) [inline lib/DDG/Goodie/Bin2Unicode.pm]: Haha, chicanery is acceptable.

**Pull request merged**